src/main/java/Deadline.java

```diff
@@ -0,0 +1,19 @@
+public class Deadline extends Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
perhaps you can remove these two lines, since inheritance of Task class, has included the initialization of the variable.